backend/src/main/java/com/woowacourse/levellog/feedback/dto/FeedbackContentDto.java

```diff
@@ -19,4 +24,12 @@ public class FeedbackContentDto {
 
     @NotNull
     private String etc;
+
+    public static FeedbackContentDto from(final Feedback feedback) {
+        return new FeedbackContentDto(feedback.getStudy(), feedback.getSpeak(), feedback.getEtc());
+    }
+
+    public Feedback toEntity(final Member member, final Levellog levellog) {
```

Comment:
```suggestion
    public Feedback toFeedback(final Member member, final Levellog levellog) {
```
이렇게 메서드명을 변경한다면 사용할 때 어떤 엔티티 객체가 리턴되는지 알기 쉽다고 생각해요. 이브의 취향대로 반영하셔도 될거 같아요.
`FeedbackContentDto.toFeedback(member, levellog);`